src/shared/lib/i18n/locales/fr/translation.json

```diff
@@ -13,35 +17,251 @@
         "description": "Choisissez le compte qui vous définit le mieux. L'accès à certaines fonctionnalités de l'application dépend du type de compte que vous sélectionnez.",
         "title": "Choisissez un type de compte"
       },
-      "vendor": "Prestataire"
-    }
+      "vendor": {
+        "type" : "Prestataire",
+        "title" : "S'authentifier",
+        "description" : "Sélectionnez l'une des options disponibles ci-dessous pour vous connecter à votre compte Échanges Entre Concepteurs."
+      }    
+    },
+    "title": "Bienvenue sur Échanges Entre Concepteurs",
+    "description": "Veuillez vous connecter pour accéder à votre compte Échanges Entre Concepteurs."
   },
-  "formatedDateTime": "{{date}} à {{time}}",
   "landing": {
-    "header": "Collaborez <br/>avec le gouvernement <br/>du Québec"
+    "header": "Collaborez <br/>avec le gouvernement <br/>du Québec",
+    "body": "Échanges Entre Concepteurs gouvernemental est une nouvelle plateforme qui contribue à bâtir un écosystème en matière d'innovation et de collaboration entre les entrepreneurs technologiques et le gouvernement du Québec.",
```

Comment:
"Échanges entre concepteurs gouvernemental" ?

Reply:
Je sais pas vraimenet la traduction exact mais, le texte original c'était : 
"The Government Digital Marketplace is a new platform that helps build an ecosystem of innovation and collaboration between technology entrepreneurs and the Quebec government."

Pareil a valider avec le PO !